Dockerfile

```diff
@@ -19,6 +19,8 @@ RUN apk update \
     && apk upgrade \
     && apk add --update --no-cache $BUILD_PACKAGES $DEV_PACKAGES $RUBY_PACKAGES
 
+RUN gem install mimemagic -v '0.3.8' --source 'https://rubygems.org/'
```

Comment:
Do we need to manually reinstall the gem? It should be installed from the Gemfile (through the `bundle install`)

Reply:
May be not, that was the message I got from the builder, (literally, copy paste) but that was before adding the dependence to alpine